performance/shopify/paginate.rb

```diff
@@ -24,7 +24,7 @@ def initialize(tag_name, markup, options)
   def render(context)
     @context = context
 
-    context.stack do
+    context.stack('paginate') do
```

Comment:
Shouldn't `'paginate'` be frozen here?
```suggestion
    context.stack('paginate'.freeze) do
```